src/main/java/com/codurance/retropolis/web/responses/BoardResponseObjectFactory.java

```diff
@@ -39,4 +40,12 @@ public BoardResponseObject create(Board board, Long userId) {
         card.getVoters().contains(userId),
         card.getVoters().size(), userService.findById(card.getUserId()).username);
   }
+
+  public List<UserBoardResponseObject> create(List<Board> boards) {
+    List<UserBoardResponseObject> responseObject = new ArrayList<>();
+    for (Board board : boards) {
+      responseObject.add(new UserBoardResponseObject(board.getId(), board.getTitle()));
```

Comment:
You can use stream here to map the object instead of using a `for` loop. 
A possible solution could be sth like below
```
return boards.stream()
        .map(board -> new UserBoardResponseObject(board.getId(), board.getTitle())).collect(toList());
```

Reply:
Thanks. Added.